analysis/extreme_events_study/visualization/README.md

```diff
@@ -0,0 +1,41 @@
+# Visualize Extreme Events
+
+## Event Display
+
+Copy the `.ig` file to local. Open the file in [iSpy WebGL](https://ispy-webgl.web.cern.ch/).
+
+## File Information
+
+* njets: 12 jets event
+* nleps: 4 leptons event with top lepton pt (553GeV)
+* pt_j: Top 2 jet pt (1.8TeV, 1.6TeV), top S_T (3.7TeV), and top H_T (3.5TeV) event
+
+## Procedure to get the `.ig` file
+
+* Modify `find_file.py` to generate a dataframe of selected event(s) from the skimmed file(s).
```

Comment:
What sort of modifications would this require?

Reply:
Would it be clearer if I change that to:

> Modify this [section](https://github.com/xinyuewu21/topcoffea/blob/fc4488f9f501b6564d87e77d2bb9f466ef79c570/analysis/extreme_events_study/visualization/find_file.py#L67-L72) of `find_file.py` to generate a dataframe of selected event(s) from the skimmed file(s).

and add the title below to `find_file.py` ?

```
########## Specify events to track ##########
code
```

Reply:
Yes, I think that explanation would be helpful, thanks.